.changeset/chilly-badgers-push.md

```diff
@@ -0,0 +1,29 @@
+---
+'@astrojs/solid-js': major
+---
+
+Render SolidJS components using [renderToStringAsync](https://www.solidjs.com/docs/latest#rendertostringasync).
```

Comment:
```suggestion
Render SolidJS components using [`renderToStringAsync`](https://www.solidjs.com/docs/latest#rendertostringasync).
```
Just checking, because I don't see these formatted as code anywhere, but seems like something that should be? If so, I'd update throughout.